VSTSAgent/VSTSAgent.psm1

```diff
@@ -243,6 +246,10 @@ function Install-VSTSAgent {
         [parameter(Mandatory = $true)]
         [uri]$ServerUrl,
 
+        [parameter(Mandatory = $false)]
+        [ValidateScript({ $_ -match "http://\d{1,3}[.]\d{1,3}[.]\d{1,3}[.]\d{1,3}:\d{1,5}" })]
```

Comment:
Can we guarantee all proxy urls will match this? E.g. what about https urls? This looks like it forces an IP to be used as well? Maybe just skip the validation?